wasmsdk/blobber.go

```diff
@@ -32,16 +32,16 @@ func listObjects(allocationId string, remotePath string) (*sdk.ListResult, error
 
 }
 
-func createDir(allocationID, remotePath string) error {
-	if len(allocationID) == 0 {
-		return RequiredArg("allocationID")
+func createDir(allocationId, remotePath string) error {
```

Comment:
We should use `ID` instead of `Id`, so let's use `allocationID`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms for why we do it.

Reply:
fixed